### DISCUSSION TITLE
translation longer than location allows: 31491 > 1311: FDBOGEKF_10000

### DISCUSSION THREAD

**Discussion by FengqingWang0712** [antismash/antismash#450] [General]
Hello there,
When I run the code "cat sample_names.txt | parallel -j2 --delay 120 "antismash --cb-general --cb-knownclusters --cb-subclusters --asf --pfam2go --skip-zip-file --cpus 16 --genefinding-tool prodigal-m {}.gbk", it showed the error "**translation longer than location allows: 31491 > 1311: FDBOGEKF_10000**". I was wondering if you know the reason. Thanks in advance.

All the best,
Fengqing

**Comment by SJShaw** [comment]
The CDS feature with locus tag (or other identifier) `FDBOGEKF_10000` has a length of 1,311 aminos and has a translation annotated that is 31,491 aminos. Either the size of that CDS is incorrect or the annotated translation is incorrect.

You can either correct the CDS to be that long or use the ambiguous ending markers for coordinates in your genbank input (e.g. `>50000` instead of just `50000`), or you can remove the translation from that CDS in the genbank and let antiSMASH regenerate it.
